spacewalk/setup/share/tomcat_java_opts.conf

```diff
@@ -1 +1 @@
-JAVA_OPTS="$JAVA_OPTS -ea -Xms256m -Xmx1G -Djava.awt.headless=true -Dorg.xml.sax.driver=com.redhat.rhn.frontend.xmlrpc.util.RhnSAXParser -Dorg.apache.tomcat.util.http.Parameters.MAX_COUNT=1024 -Dnet.sf.ehcache.skipUpdateCheck=true"
+JAVA_OPTS="$JAVA_OPTS -ea -Xms256m -Xmx1G -Djava.awt.headless=true -Dorg.xml.sax.driver=com.redhat.rhn.frontend.xmlrpc.util.RhnSAXParser -Dorg.apache.tomcat.util.http.Parameters.MAX_COUNT=1024 -Dnet.sf.ehcache.skipUpdateCheck=true $JMX_TOMCAT_OPTS $JMX_TASKOMATIC_OPTS"
```

Comment:
I think this is wrong here. Tomcat should expose its metrics on port 5556, Taskomatic on 5557. This is not shared.